sonic_platform_base/sonic_storage/usb.py

```diff
@@ -0,0 +1,140 @@
+#
+# usb.py
+#
+# Implementation of SSD Utility API for eUSB.
+# It reads eUSB health, model, firmware, and serial from /sys/block/*.
+#
+
+try:
+    import os
+    from .storage_common import StorageCommon
+    from sonic_py_common import syslogger
+    from blkinfo import BlkDiskInfo
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
+
+
+NOT_AVAILABLE = "N/A"
+
+class UsbUtil(StorageCommon):
+
+    model = NOT_AVAILABLE
+    serial = NOT_AVAILABLE
+    firmware = NOT_AVAILABLE
+    temperature = NOT_AVAILABLE
+    health = NOT_AVAILABLE
+    vendor = NOT_AVAILABLE
+    disk_io_reads = NOT_AVAILABLE
+    disk_io_writes = NOT_AVAILABLE
+    reserved_blocks = NOT_AVAILABLE
+    blkd = {}
+
+
+    def __init__(self, diskdev):
+
+        self.diskdev = diskdev
+        self.path = os.path.join('/sys/block', os.path.basename(diskdev))
+        StorageCommon.__init__(self, diskdev)
+
+        self.log_identifier = "UsbUtil"
+        self.log = syslogger.SysLogger(self.log_identifier)
```

Comment:
@assrinivasan seems like log is not used

Reply:
removed in latest commit